### PR TITLE
add new wgannon bio

### LINKED ISSUE #64: Add user wgannon
blah blah blah

### PR DESCRIPTION
**Pull request by wgannon-sjrwmd**
I created this file because the voice in the box said to

Resolves #64 
@githubteacher - lost again

@skempins - this is really easy

### CODE REVIEW

**Comment by wgannon-sjrwmd** [comment]
blah blah blah

File: bios/wgannon.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+##yea - i'm slow
+
+###blah blah blah
+- NONYA
```
Comment (jpallepo) [inline bios/wgannon.md:4]: add comments..

**Comment by jpallepo** [comment]
:+1

**Comment by wgannon-sjrwmd** [comment]
Failure

**Pull request merged**